src/login/checkSettingsOnStartup.ts

```diff
@@ -0,0 +1,84 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { IActionContext } from "@microsoft/vscode-azext-utils";
+import { commands, ExtensionContext, window } from "vscode";
+import { localize } from "../utils/localize";
+import { getSettingValue } from "../utils/settingUtils";
+import { AzureAccountLoginHelper } from "./AzureLoginHelper";
+import { cachedSettingKeys, SettingsCache, settingsCacheKey, SettingsCacheVerified } from "./SettingsCache";
+
+export async function checkSettingsOnStartup(extensionContext: ExtensionContext, actionContext: IActionContext, loginHelper: AzureAccountLoginHelper): Promise<void> {
+	const numSettings = cachedSettingKeys.length;
+
+	const lastSeenSettingsCache: SettingsCache | undefined = extensionContext.globalState.get(settingsCacheKey);
```

Comment:
nit: `    ` instead of `\t`

Reply:
Good catch - I turned my auto formatter off a while ago because it was causing problems. And yet there are still problems 😄